server/catalog/table.cpp

```diff
@@ -90,9 +90,8 @@ velox::RowTypePtr BuildPkType(const std::vector<Column>& columns,
   types.reserve(pk_columns.size());
 
   for (auto pk_col_id : pk_columns) {
-    auto it = std::find_if(
-      columns.begin(), columns.end(),
-      [pk_col_id](const Column& col) { return col.id == pk_col_id; });
+    auto it = absl::c_find_if(
+      columns, [pk_col_id](const Column& col) { return col.id == pk_col_id; });
```

Comment:
NIT:
```suggestion
      columns, [&](const Column& col) { return col.id == pk_col_id; });
```
it's more logical, because you want always inline